pyproject.toml

```diff
@@ -64,7 +64,7 @@ Werkzeug = "2.3.7"
 MarkupSafe = "2.1.5"
 # REVIEW: v2 is using sha512 instead of sha1 by default (in v1)
 itsdangerous = "2.1.2"
-notifications-utils = { git = "https://github.com/cds-snc/notifier-utils.git", tag = "52.2.2" }
+notifications-utils = { git = "https://github.com/cds-snc/notifier-utils.git", branch = "reinstate-bulk-sms-limit" }
```

Comment:
Is this an exception here to deviate from the regular tag versioning?

Reply:
It's just a temporary measure, until the corresponding utils PR is merged, so that this PR can be tested locally without too much extra work.